media-gfx/photoqt/photoqt-4.7.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+ECM_KDEINSTALLDIRS=false
+KFMIN=6.5.0
+inherit ecm optfeature python-single-r1 toolchain-funcs
+
+DESCRIPTION="Simple but powerful Qt-based image viewer"
+HOMEPAGE="https://photoqt.org/"
+SRC_URI="https://photoqt.org/downloads/source/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="barcode chromecast devil exif freeimage geolocation graphicsmagick +imagemagick lcms mpv pdf raw vips"
+REQUIRED_USE="
+	?? ( imagemagick graphicsmagick )
```

Comment:
Please avoid this -- we try to do "provider USE", see https://bugs.gentoo.org/314431#c28. So we don't want the REQUIRED_USE there. We want USE=imagemagick to control the "IM-style functionality" and USE=graphicsmagick to control which provider of that is used.

Reply:
Thanks. It's clear to me now that previous ebuild was ok ...